receiver/mysqlreceiver/testdata/scraper/expected.yaml

```diff
@@ -1666,7 +1666,7 @@ resourceMetrics:
                   startTimeUnixNano: "1644862687825728000"
                   timeUnixNano: "1644862687825772000"
               isMonotonic: true
-            unit: "s"
+            unit: s
```

Comment:
This appears unrelated

Reply:
This is the only change from running golden test on all the receivers.

Reply:
I think it's unrelated to these changes but it's trivial so we'll just pick it up.